packages/flutter/lib/src/material/slider.dart

```diff
@@ -672,7 +672,7 @@ class _RenderSlider extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
        _textDirection = textDirection {
     _updateLabelPainter();
     final GestureArenaTeam team = GestureArenaTeam();
-    _drag = HorizontalDragGestureRecognizer()
+    _drag = HorizontalDragGestureRecognizer(touchSlop: 0)
```

Comment:
As @Hixie mentioned `touchSlop` should be `> 0` something like `8` or `kTouchSlop/2` to avoid changing slider when scrolling page, list, etc
Also make this change in `CupertinoSlider`.

Reply:
As I mentioned in the issue,

In flutter_gallery, the range sliders are inside a Column. There is no scrolling factor, yet this triggers the issue.

Additionally, regarding the slop, even in ListView, I've never seen this in other platforms like Android or iOS. Shouldn't it be optional?

Reply:
@Hixie, if Android and iOS have this behavior, shouldn't this be the default behavior of Material`Slider` and `CupertinoSlider`?

Reply:
![GIF-191213_131837](https://user-images.githubusercontent.com/351125/70814896-312f0680-1dab-11ea-8a76-bd14a2695a3e.gif)

Android settings app

Reply:
When I test Android here it behaves as I would expect Flutter to behave (there's a slop in both the horizontal and vertical directions).

But that's not the issue here. The issue is that this used to work, and then it regressed (according to the issue). We need to bisect to find when it regressed. The best way to do that is to create a unit test that exactly reproduces the issue you're concerned with, verify that it worked in the past, and then use `git bisect` to automatically determine what commit broke it.

Reply:
What is the expected behavior of the horizontal slider when it is in a container that has horizontal scroll (such as a TabView)?

Before the RangeSlider was added to the Gallery (https://github.com/flutter/flutter/commit/04bd87794d0704096b70308f7ecb52f1a92c62b1), the Slider demo was not in a TabView, and moved on short distances. Once the RangeSlider was added, a TabView was added for Slider and RangeSlider demo to be on separate tabs, and then the issues started surfacing.

In other words, this issue seemed to have always existed, but was only surfaced in the gallery once the Slider demo was put into a container with horizontal scroll.